docs/cadence/language/values-and-types.mdx

```diff
@@ -925,6 +925,21 @@ let c = str[0] // is the Character "a"
 
 `Character` values can be converted into `String` values using the `toString` function:
 
+-
+    ```cadence
+    let utf8: [UInt8]
+    ```
+
+    The byte array of the UTF-8 encoding
+
+    ```cadence
+    let a: Character = "a"
+    let a_bytes = a.utf8 // `a_bytes` is `[97]`
+
+    let bouquet: Character = "\u{1F490}"
+    let bouquet_bytes = bouquet.utf8 // `bouquet_bytes` is `[240, 159, 146, 144]`
+    ```
+
```

Comment:
I guess this should be moved to the bottom of `Character Fields and Functions` section. 
The sentence above describes the `toString` function, and the `toString` function's example is below this newly added part.

Reply:
Thanks for catching that. Missed it completely. Updated the PR.